sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -188,7 +189,10 @@ public override bool IsDisposed
             IWebProxy proxy,
             bool useSingleSession,
             TimeSpan operationTimeout,
-            ServiceBusTransportMetrics metrics)
+            ServiceBusTransportMetrics metrics,
+#pragma warning disable CA1801 // Review unused parameters. This parameter will be used for future planned features.
+            string identifier = default)
```

Comment:
I would just remove this until we actually need it. Who knows - that day may never actually come and we would be left with this odd parameter.